crates/context/config/src/stellar/stellar_types.rs

```diff
@@ -22,34 +26,83 @@ pub struct StellarApplication {
     pub metadata: Bytes,
 }
 
+impl<'a> FromWithEnv<Application<'a>> for StellarApplication {
+    fn from_with_env(value: Application<'a>, env: &Env) -> Self {
+        StellarApplication {
+            id: BytesN::from_array(env, &value.id.rt().expect("infallible conversion")),
+            blob: BytesN::from_array(env, &value.blob.rt().expect("infallible conversion")),
+            size: value.size,
+            source: String::from_str(env, &value.source.0.into_owned()),
+            metadata: Bytes::from_slice(env, &value.metadata.0.as_bytes().to_vec()),
+        }
+    }
+}
+
+impl<'a> From<StellarApplication> for Application<'a> {
+    fn from(value: StellarApplication) -> Self {
+        // Convert Soroban String to std::String
+        let mut bytes = vec![0u8; value.source.len() as usize];
```

Comment:
Couldn't find a way to optimize as soroban string didn't have needed methods and other solution was pretty much same amount of code and no visible improvement code + execution

Reply:
Works

Reply:
Actually when updated throw in CI this error so reverted:
<img width="1100" alt="Screenshot 2025-01-30 at 14 27 54" src="https://github.com/user-attachments/assets/93271de6-ab50-4968-8f0f-fa2ec1688d11" />